packages/mdc-textfield/foundation.js

```diff
@@ -150,4 +156,20 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       disabled: false,
     };
   }
+
+  registerAutoCompleteHandler_(input) {
+    return new Promise((resolve, reject) => {
+      let hasKeyInteraction = false;
+
+      input.addEventListener('input', (e) => {
+        if (!hasKeyInteraction) {
+          resolve(e);
+        }
+      });
+
+      input.addEventListener('keydown', function(e) {
```

Comment:
Do we need to have the event object passed into this function? It doesn't look like we use it.

Also, this should be a fat-arrow function.

```js
('keydown', () => {
  hasKeyInteraction = true;
});
```